src/components/Books.js

```diff
@@ -0,0 +1,83 @@
+const Books = () => {
+  const booksProperties = [
+    {
+      id: 1,
+      category: 'Action',
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+      percent: '64%',
+      chapter: 'Chapter 17',
+    },
+    {
+      id: 2,
+      category: 'Science Fiction',
+      title: 'Dune',
+      author: 'Frank Herbert',
+      percent: '8%',
+      chapter: 'Chapter 3: A Lesson Learned',
+    },
+    {
+      id: 3,
+      category: 'Economy',
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+      percent: '0%',
+      chapter: 'Introduction',
+    },
+
+  ];
+  return (
+    <div className="mainContainer">
+      <div className="bookContainer">
+        <ul className="BookUL">
+          {booksProperties.map((booksProperty) => (
+            <li key={booksProperty.id} className="bookList">
+              <div className="book">
+                <div className="bookInfo">
+                  <h4 className="bookCategory">{booksProperty.category}</h4>
+                  <h2 className="BookTitle">{booksProperty.title}</h2>
+                  <h6 className="bookAuthor">{booksProperty.author}</h6>
+                  <div className="actionBtn">
+                    <button type="button" className="crudBtn">Comment</button>
+                    <div className="Vdivider" />
+                    <button type="button" className="crudBtn">Remove</button>
+                    <div className="Vdivider" />
+                    <button type="button" className="crudBtn">Edit</button>
+                  </div>
+                </div>
```

Comment:
- [ ] Nice work you did her 👏 . However, you're meant to use a component based approach to list books add the form which will all be inside this component. Find a detailed guide below.

- [ ] This whole  `Books` component, should contain the following components:

- `BookList` (component)
-- Renders a list of books using the `Book` component
- `Book` (component)
-- Takes the book data as a prop and renders the title and the author
--- It should have  a button for removing books. The button wouldn't remove the books yet, this is just a placeholder for now

- `NewBookForm`
--- Renders a form with inputs for the title and the author of the book, it has a button to submit. The button wouldn't add the book yet, this is just a placeholder for now

### instructions:
![image](https://github.com/MozamelJawad/bookstore/assets/50990925/31d3d651-7de5-4d44-8ee7-932b90712c8b)